<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FNAF</class>
 <widget class="QMainWindow" name="FNAF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_fundo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../imagens/img.qrc">:/img/lab.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_esq">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>282</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_cima">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_dir">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>280</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_papel3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>480</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_papel1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_papel2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>110</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>150</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>200</number>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>110</width>
       <height>150</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>150</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../imagens/img.qrc">:/img/estou_obs.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>190</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>110</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>190</width>
      <height>110</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>-4</x>
       <y>0</y>
       <width>201</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../imagens/img.qrc">:/img/asdfasdf.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>190</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>110</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>190</width>
      <height>110</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../imagens/img.qrc">:/img/nao_estamos_parados.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bt_frames_lab">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>821</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>lb_fundo</zorder>
   <zorder>bt_esq</zorder>
   <zorder>bt_cima</zorder>
   <zorder>bt_dir</zorder>
   <zorder>bt_papel3</zorder>
   <zorder>bt_papel1</zorder>
   <zorder>bt_papel2</zorder>
   <zorder>bt_frames_lab</zorder>
   <zorder>frame3</zorder>
   <zorder>frame2</zorder>
   <zorder>frame1</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../imagens/img.qrc"/>
 </resources>
 <connections/>
</ui>
